src/multiharp_toolkit/ptu_parser.py

```diff
@@ -98,12 +99,17 @@ def parse_record(self, data: int):
             truetime = self.oflcorrection + timetag
             self.append_events(channel + 1, truetime)
 
-    def append_events(self, channel: int, timestamp: float):
+    def timetag2time(self, timetag: int) -> int:
```

Comment:
Function names should start with verbs, such as "convert." They should not use non-standard grammar like "2" for "to."

"Time" is too vague. What is this value? Is it a relative timestamp?

```suggestion
    def convert_timetag_to_relative_timestamp(self, timetag: int) -> int:
```

Reply:
Thank you for comment.

>"Time" is too vague. What is this value? Is it a relative timestamp?

Sorry, I am not sure..
I will check.

I think it is relative timestamp, but, I do not know from what the relative timestamp is.

Reply:
I might overlooked it, but I couldn't find the clear definition of "relative timestamp".

>But this is probably good enough for now.

I think so.
So the method name you suggested is good.